tokio-util/Cargo.toml

```diff
@@ -46,6 +46,7 @@ tracing = { version = "0.1.25", default-features = false, features = ["std"], op
 
 [target.'cfg(tokio_unstable)'.dependencies]
 hashbrown = { version = "0.14.0", optional = true }
+ahash = { version = "0.8.7", optional = true }
```

Comment:
Instead of adding such a hack, we might want to disable the default features of hashbrown and remove the ahash from the dependency tree.

Reply:
Let me put that in its own PR so it gets a changelog entry. #6541